examples/platforms/nrf52811/radio.c

```diff
@@ -733,17 +729,15 @@ void nrf_802154_received_raw(uint8_t *p_data, int8_t power, uint8_t lqi)
         receivedFrame->mInfo.mRxInfo.mAckedWithFramePending = false;
     }
 
-#if OPENTHREAD_CONFIG_TIME_SYNC_ENABLE
-    // Get the timestamp when the SFD was received.
-    uint32_t offset =
-        (int32_t)otPlatAlarmMicroGetNow() - (int32_t)nrf_802154_first_symbol_timestamp_get(time, p_data[0]);
-    receivedFrame->mInfo.mRxInfo.mTimestamp = otPlatTimeGet() - offset;
-#else
+#if !OPENTHREAD_CONFIG_TIME_SYNC_ENABLE
```

Comment:
I don't think we need this any more. We should always fill the timestamp field.

Reply:
Sure.